tools/sharkbait/project.clj

```diff
@@ -15,11 +15,15 @@
   :manifest {"Git-Ref" ~(git-ref)}
   :uberjar-name "sharkbait-standalone.jar"
   :dependencies [[edu.internet2.middleware.grouper/grouper "2.2.1"]
+                 [honeysql "0.6.2"]
+                 [korma "0.4.2"]
                  [net.sf.ehcache/ehcache "2.10.1"]
                  [org.clojure/clojure "1.7.0"]
                  [org.clojure/tools.logging "0.3.1"]
                  [org.hibernate/hibernate-core "3.6.10.Final"]
                  [org.hibernate/hibernate-ehcache "3.6.10.Final"]
-                 [org.iplantc/common-cli "5.2.4.0"]]
+                 [org.iplantc/common-cli "5.2.4.0"]
+                 [org.iplantc/kameleon "5.2.4.0"]
+                 [postgresql "9.3-1102.jdbc41"]]
   :main sharkbait.core
   :profiles {:uberjar {:aot :all}})
```

Comment:
The changes to sharkbait all support one of two things. The first is to namespace the folders in Grouper so that multiple DE environments can share one Grouper instance. The second, more extensive, change is to register existing DE apps in Grouper.